.github/workflows/bindings_nodejs.yml

```diff
@@ -199,6 +199,9 @@ jobs:
       - name: Move artifacts
         run: yarn artifacts
 
+      - name: Build to generate main files
+        run: yarn build
```

Comment:
This is not very good here, we should put index.d.ts and generated.js into artifacts in any previous step.